### ISSUE TITLE
General Status Update

### ISSUE THREAD

**Issue by adbr8389** [sinara-hw/Shuttler#26]
@gkasprow @tprzywoz we (Ions storage) are curious if there any updates regarding the status of the shuttler design

**Comment by gkasprow** [comment]
That's a question to @kaolpr. He produces it currently.

**Comment by kaolpr** [comment]
We're expecting to have prototypes in ca 6 weeks. After that we'll need some time (+ca 4 weeks) to test and verify hardware and firmware.

**Comment by adbr8389** [comment]
Great, thanks for the response.

**Comment by adbr8389** [comment]
@kaolpr I am getting some impression of the status of the shuttler prototyping from the issues - seems like progress is being made but things are still firmly in the prototype and debug stage. Could you quickly give me a (short) coherent story of what is happening on your end? Thanks!

**Comment by kaolpr** [comment]
Well, we do have v1.0 up and running. At TechnoSystem we used Digilent Genesys 2 with some minimal sample lookup table based firmware to evaluate the output signal. 

Some hardware issues were identified and already fixed for v1.1 (https://github.com/sinara-hw/Shuttler/issues/30). Beside that, an issue with bandwidth was identified (https://github.com/sinara-hw/Shuttler/issues/36). Hope to have a decision on fix / nofix soon (you're welcome to contribute). 

We did not test AFE at TechnoSystem yet.

Ongoing efforts:

- AFE testing + resolving encountered issues
- finalize v1.1
- manufacture v1.1.

We're hoping to start v1.1 manufacturing in ca 3 weeks from now.
